src/printer.js

```diff
@@ -1390,6 +1390,13 @@ function genericPrintNoParens(path, options, print) {
       var expressions = path.map(print, "expressions");
 
       function removeLines(doc) {
+        // Exceptions returning raw doc.
+        if (
+          n.expressions && n.expressions.length === 1 &&
+          n.expressions[0].type === "ObjectExpression"
```

Comment:
This is way too specific, if you put it inside of another expression it's also going to break. I don't think that we should fix this issue here but find the root cause and fix it.